cert-manager/README.md

```diff
@@ -0,0 +1,12 @@
+To bootstrap (per the [cert-manager getting started
+guide](https://cert-manager.readthedocs.io/en/latest/getting-started/install.html#installing-with-regular-manifests)):
+
+```
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value core/account)
+kubectl apply -f cert-manager.yaml --validate=false
```

Comment:
nit: possibly add a note mentioning that `--validate=false` can go away with k8s 1.13+

Reply:
done